extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitter.java

```diff
@@ -77,12 +80,28 @@ public void start()
       } else {
         log.error("HTTPServer is already started");
       }
-    } else if (strategy.equals(PrometheusEmitterConfig.Strategy.pushgateway)) {
-      pushGateway = new PushGateway(config.getPushGatewayAddress());
+    } else if (strategy.equals(PrometheusEmitterConfig.Strategy.pushgateway) && config.getPushGatewayAddress() != null) {
```

Comment:
Nit: can we move this check as a Precondition check in PrometheusEmitterConfig for the case when strategy is set to pushGateway but the gateway address is null ?

Reply:
I implemented your comment, thanks for the advice

Reply:
Is there any default value you think we should insert in case the address is null?

Reply:
Thanks for the comment, fixed 👍